src/components/MyAccount/Settings/SaveCancelButtons.tsx

```diff
@@ -4,7 +4,7 @@ type SaveCancelButtonProps = {
   isDisabled?: boolean
   onCancel: () => void
   onSave: () => void
-  inputType?: "phones" | "emails"
+  inputType?: string
```

Comment:
was there a change lost here? i remember there being some conditional logic in the saveCancelButton component around this. I think we discussed it before, but I think it would be ok to add the extra `inputType="password"` in `PasswordForm.tsx`